config/default.yml

```diff
@@ -3898,6 +3898,12 @@ Style/UnlessElse:
   Enabled: true
   VersionAdded: '0.9'
 
+Style/UnnecessaryArguments:
+  Description: 'Check for unnecessary arguments to builtin methods.'
+  Enabled: true
```

Comment:
Can you specify `Enabled: pending` instead of `Enabled: true`.

Reply:
Changed to `pending`.